meinberlin/apps/account/templates/meinberlin_account/profile.html

```diff
@@ -4,10 +4,9 @@
 
 {% block title %}{% trans 'Your user profile' %} &mdash; {{ block.super }}{% endblock %}
 {% block dashboard_content %}
-<h1 class="u-first-heading">{% trans 'Your profile' %}</h1>
+    <h1 class="u-first-heading">{% trans 'Your profile' %}</h1>
 
-<div class="general-form">
```

Comment:
`class=general-form` is also used in 'meinberlin/templates/socialaccount/connections.html'